src/main/java/bio/terra/workspace/service/job/JobBuilder.java

```diff
@@ -54,14 +54,15 @@ public JobBuilder addParameter(String keyName, Object val) {
   }
 
   // submits this job to stairway and returns the jobId immediately
-  public String submit() {
-    return jobServiceRef.submit(flightClass, jobParameterMap, jobId);
+  public String submit(boolean duplicateFlightOk) {
```

Comment:
Let's fix the comment to be javadoc on this method and explain what `boolean duplicateFlightOk` means. Ditto on submitAndWait.
If this is truly very short term until the next PR, we can leave this as a boolean. Otherwise, consider making an enum to make this readable at callsites and easier to document. Boolean parameters should be used exceedingly sparingly. go/java-practices/boolean-parameters
A TODO to move this into the JobBuilder would make me happy.

Reply:
Fixed javadoc and added TODO, I'd rather handle this more cleanly in JobBuilder (might still need an enum as part of that change, tbd).